hnn_core/tests/test_gui.py

```diff
@@ -103,7 +103,8 @@ def check_drive(drive1, drive2, keys):
                             'connectivity', 'copy', 'gid_to_type',
                             'plot_cells', 'set_cell_positions',
                             'to_dict', 'write_configuration',
-                            'external_drives', 'external_biases'])
+                            'external_drives', 'external_biases',
+                            'update_weights'])
```

Comment:
This is similar to the change in the __eq__ method of the Network class. This is a custom eq function used just for checking GUI rebuilt networks vs API generated networks.